requirements.txt

```diff
@@ -4,3 +4,4 @@ websockets
 brotli; platform_python_implementation=='CPython'
 brotlicffi; platform_python_implementation!='CPython'
 certifi
+Box
```

Comment:
Please explain

Reply:
dot notation:

```python
from box import Box

movie_box = Box({ "Robin Hood: Men in Tights": { "imdb stars": 6.7, "length": 104 } })

movie_box.Robin_Hood_Men_in_Tights.imdb_stars
```

more here: https://github.com/cdgriffith/Box/

i can remove that if it's too much of a trouble. (i see CI's are failing because of that)

also: please don't merge, some minor changes are coming, e.g. upstream bug for endpoint (returns 500): https://api.lib.harvard.edu/v2/collections/{collectionId}/